cfme/infrastructure/provider/__init__.py

```diff
@@ -333,7 +333,7 @@ def step(self):
     def resetter(self):
         """Reset view and selection"""
```

Comment:
Should this be updated to describe which view it is resetting to?

Reply:
sure